src/gui/plugins/component_inspector/ComponentInspector.cc

```diff
@@ -110,7 +110,7 @@ using namespace gazebo;
 
 //////////////////////////////////////////////////
 template<>
-void ignition::gazebo::setData(QStandardItem *_item,
+void gazebo::setData(QStandardItem *_item,
```

Comment:
We're `using namespace gazebo` here, but this wasn't updated. It would be nice to get rid of those which will need to be changed to `sim` on version 7

Reply:
It broke on just `setData` unfortunately.. I forgot why, but the compiler throws a fit